package-lock.json

```diff
@@ -8,9 +8,15 @@
       "name": "instagram-bootcamp",
```

Comment:
you can probably add the package-lock.json to the .gitignore as everyone will have their own package-lock.json based on their local dependency installs